### DISCUSSION TITLE
Fatal Error: db_cxx.h

### DISCUSSION THREAD

**Discussion by SireWilliamson** [btcpayserver/btcpayserver#2191] [Technical Support] [answered]
I'm trying to combile the BCT Core, and I get this error when I run the "make" command.
We are trying to setup with apache and mysql, because that is what all our other systems use. 
Is mysql an issue? Or this error something different entirely?

**Comment by SireWilliamson** [comment]
I cut the end of the error of sorry.
fatal error: db_cxx.h: No such file or directory
   22 | #include <db_cxx.h>
      |          ^~~~~~~~~~

**Reply by SireWilliamson** [reply]
The files is in these locations
~/bitcoin/db4/db-4.8.30.NC/build_unix/db_cxx.h
~/bitcoin/db4/db-4.8.30.NC/build_vxworks/db_cxx.h
~/bitcoin/db4/db-4.8.30.NC/build_windows/db_cxx.h
~/bitcoin/db4/db-4.8.30.NC/docs/programmer_reference/upgrade_3_0_db_cxx.html
~/bitcoin/db4/docs/programmer_reference/upgrade_3_0_db_cxx.html
~/bitcoin/db4/include/db_cxx.h

**Comment by Kukks** [comment, marked as answer]
This has nothing to do with btcpay server.  Message the bct core community.

**Reply by SireWilliamson** [reply]
Ok.
